### ISSUE TITLE
Add original unit name subtitle

### ISSUE THREAD

**Issue by AdamLay** [AdamLay/opr-army-forge#378]

**Comment by BaptisteD** [comment]
@AdamLay could you describe what's the user need behind this task ?  :)

**Comment by AdamLay** [comment]
Hi @BaptisteD ! Yes @klauskopec has basically nailed it there, although we also have another reason.
Many heroes in the games (all of them?) have an upgrade section that allows you to essential transform your base hero into a specialist one. Let's say a HDF "Company Leader" can become a "Forward Observer". We're introducing a flag on the upgrade section (like isHeroUpgrade) that will allow us to specifically target these upgrades. One of the things we can do with this, is reintroduce the "flavour" by auto renaming the base model hero the specific type automatically when these upgrades are selected. We don't want to lose the original name though, so it makes sense to still display this.
Also as @klauskopec was saying we have the same thing for custom units - it can be useful to you and perhaps your opponent to know that "Mr Super Boss Smasher" is actually a "Champion".
Thanks!

**Comment by AdamLay** [comment]
![image](https://user-images.githubusercontent.com/4223668/201885158-34ba6cff-31a4-489b-b67a-5b8cb881fe1b.png)
As an example - something like this?

**Comment by klauskopec** [comment]
Looks good. This could be conditional to only show if the name was changed.

Will you also add it to the printed view? Maybe with the option to toggle it on and off for those who want the cleaner look? That'd be great.

**Comment by BaptisteD** [comment]
I was going to ask that also. 
Where does this information bring value ? 
Everywhere, in the card view, only in the edit view ?
I will be tempted to say everywhere.

**Comment by AdamLay** [comment]
Yes agreed - I am also leaning towards everywhere!
For the list view we could consider a format like `Priestess (High Sister)` and then everywhere else use the subtitle approch?
Thanks,

**Comment by klauskopec** [comment]
Sounds good as proposed.

**Comment by AdamLay** [comment]
![image](https://user-images.githubusercontent.com/4223668/201900932-f5a30271-c664-480a-b469-c1ac305a77b4.png)
![image](https://user-images.githubusercontent.com/4223668/201901700-fb4b3263-8239-4da8-aeed-f2d4e3cf1e43.png)

I wonder if the card view would be better with a subtitle - struggling to fit it in though :)

**Comment by klauskopec** [comment]
I agree, the subtitle will add significantly to the height of each card. On the other hand, the solution you have with brackets with the original name will very soon lead to a line break/wrap in that box, pushing the model count, cost and - if available - campaign XP to the next line. I know because that solution is literally what I've done so far manually ;)

**Comment by BaptisteD** [comment]
I would argue that we should stay consistent across the app.
So if we use subtitle nearly everywhere, it should also be the case in the list and card view.

**Comment by BaptisteD** [comment]
In the other case, in the card view I would at least use the same style as the unit's point cost. 
And in the list view the same as the unit's model count [1]
